lms/djangoapps/teams/tests/test_views.py

```diff
@@ -540,6 +542,10 @@ def test_full_student_creator(self):
         team_membership = team['membership']
         del team['membership']
 
+        # verify that it's been set to a time today.
+        self.assertEqual(parser.parse(team['last_activity_at']).date(), datetime.today().date())
```

Comment:
That's a good point. I figured a day wouldn't matter, but I could see it breaking if you're trying to run tests at like 8pm or something like that.